prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -278,8 +289,9 @@ export class CollegeCertificationFormComponent implements OnInit {
     } else {
       licenseNumberValidators.push(FormControlValidators.alphanumeric);
     }
-    this.formUtilsService.setValidators(this.licenseNumber, licenseNumberValidators);
-
+    if (this.collegeCode.value === CollegeLicenceClassEnum.CPSBC) {
+      this.formUtilsService.setValidators(this.licenseNumber, licenseNumberValidators);
+    }
```

Comment:
Why are the validators set only when CPSBC?

Reply:
Good point. The logic is updated so that it will add the validation to the license number control only if it is not in condensed mode and the college code is BCCNM - because the license number is not showing for BCCNM in remote access mode .